### PR TITLE
Fix: Compose menu icons are visible even in unexpanded state.

### PR DESCRIPTION
**Pull request by jainkuniya**
Fix: #2979

### CODE REVIEW

File: src/styles/composeBoxStyles.js [hunk 1]
```diff
@@ -67,6 +67,7 @@ export default ({ color, backgroundColor, borderColor }: Props) => ({
   composeMenu: {
     flexDirection: 'row',
     overflow: 'hidden',
+    backgroundColor: 'transparent',
```
Comment (borisyankov) [inline src/styles/composeBoxStyles.js]: The style is very weird (as I mentioned in the last PR review :) )
I'd add a comment here so we don't accidentally remove it again.

Comment (jainkuniya) [reply src/styles/composeBoxStyles.js]: @borisyankov Added comment :) 👍

File: src/styles/composeBoxStyles.js [hunk 2]
```diff
@@ -67,6 +67,9 @@ export default ({ color, backgroundColor, borderColor }: Props) => ({
   composeMenu: {
     flexDirection: 'row',
     overflow: 'hidden',
+    // this backgroundColor style is a test and try hack to fix #2979,
```
Comment (borisyankov) [inline src/styles/composeBoxStyles.js]: Maybe it is better to know why this happens and if this hack works reliably? :)

**Comment by gnprice** [comment]
:smile: Based on the comment in the new version it sounds like this is not something we should merge.

I definitely would be interested in a fix for #2979, though! Seems like some more debugging is required. :wink:

**Pull request closed without merging**